src/OpenSage.Game/Graphics/Cameras/Camera.cs

```diff
@@ -80,9 +80,11 @@ public float NearPlaneDistance
 
         public Camera(Func<Viewport> getViewport)
         {
+
             _getViewport = getViewport;
             _viewport = getViewport();
 
+            View = Matrix4x4.Identity;
```

Comment:
Why was this necessary?

Reply:
Sometimes the View matrix was not initialized resulting in crashes with NaN / division by zero. This gets initialized from outside during a tick, but apparently the order seems to be different for networking. However visually everything is still the same